src/preamble.js

```diff
@@ -892,6 +892,37 @@ function getBinaryPromise() {
   });
 }
 
+#if SOURCE_MAPS
+var wasmSourceMapFile = '{{{ WASM_BINARY_FILE }}}.map';
+if (!isDataURI(wasmBinaryFile)) {
+  wasmSourceMapFile = locateFile(wasmSourceMapFile);
+}
+
+function getSourceMap() {
+  try {
+    if (Module['wasmSourceMap']) return Module['wasmSourceMap'];
```

Comment:
better to avoid adding a new `Module` property, unless there is a strong reason?